crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -4,7 +4,11 @@ use bevy_ecs::{
     reflect::{AppTypeRegistry, ReflectComponent, ReflectMapEntities},
     world::World,
 };
+<<<<<<< HEAD
```

Comment:
:eyes:

Reply:
yes i really mucked up this merge :facepalm: